Dockerfile.selenium

```diff
@@ -4,7 +4,7 @@ ENV PYTHONUNBUFFERED 1
 USER root
 RUN apt-get update && apt-get install -yq python3.7 python3-pip
 RUN pip3 install --upgrade pip
-RUN pip3 install selenium psycopg2-binary==2.8.6 pyyaml==5.4.1 Pillow==8.3.1
+RUN pip3 install selenium psycopg2-binary==2.8.6 pyyaml==5.4.1 Pillow==8.3.2

```

Comment:
why is this pillow upgrade being included here?

Reply:
@JFU-GIT Also curious about this?  Are there errors if it's removed?

Reply:
When Pillow is bumped, this dockerfile left unchanged - so selenium tests would fail.

Reply:
@JFU-GIT Was mainly wondering why Pillow is needed here vs. installed for the web service normally from the requirements?

Is it required for Selenium?   This looks OK, if so.